generic/tests/cfg/vlan.cfg

```
@@ -23,9 +23,8 @@
         vlan_num = 1
         win_vlan_id = 900
         cdroms += " virtio"
-        prepare_netkvmco_cmd = 'xcopy %s c:\\ /y && rundll32 netkvmco.dll,RegisterNetKVMNetShHelper'
-        set_vlan_cmd = 'netsh netkvm setparam 0 param=vlanid value=${win_vlan_id}'
         driver_verifier = netkvm
+        set_vlan_cmd = 'powershell -command "Set-NetAdapter -Name '%s' -VlanID ${win_vlan_id} -Confirm:$False"'
```

I mean the outermost `'`.
```suggestion
        set_vlan_cmd = powershell -command "Set-NetAdapter -Name '%s' -VlanID ${win_vlan_id} -Confirm:$False"
```

---

Sorry for misunderstood your intention， updated, please help review it again.